libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java

```diff
@@ -382,6 +393,23 @@ public void startNavigation(@NonNull DirectionsRoute directionsRoute, @NonNull D
    */
   public void stopNavigation() {
     Timber.d("MapboxNavigation stopNavigation called");
+    enableFreeDrive();
```

Comment:
This method shouldn't be invoked by us. We should only do something along the lines of:
```
if (isFreeDriveConfigured) {
  freeDriveLocationUpdater.start()
}
```
but then, `disableFreeDrive()` method should reset `isFreeDriveConfigured` back to `false`. Or perhaps another state variable is required, depending how you'd like to approch it.